src/components/graphs/LongTermAveragesGraph/LongTermAveragesGraph.js

```diff
@@ -0,0 +1,160 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Row, Col } from 'react-bootstrap';
+
+import _ from 'underscore';
+
+import TimeOfYearSelector from '../../Selector/TimeOfYearSelector';
+import DataGraph from '../DataGraph/DataGraph';
+import ExportButtons from '../ExportButtons';
+
+import {
+  blankGraphSpec,
+  displayError,
+  noDataMessageGraphSpec,
+  shouldLoadData,
+} from '../graph-helpers';
+import {
+  timeKeyToResolutionIndex,
+  validateLongTermAverageData,
+} from '../../../core/util';
+import { getData } from '../../../data-services/ce-backend';
+import { exportDataToWorksheet } from '../../../core/export';
+
+// This component renders a graph, over actual time points (as opposed to the
+// "average year" time points of an annual cycle graph), of spatially averaged
+// values of long-term average data for the selected model, variable, and
```

Comment:
This description is a little confusing to me. The Long Term Average graph shows a series of averages for subsequent (and sometimes overlapping) climatological periods. I wouldn't say it represents "actual time points" any more than the annual cycle graph, which also shows average values from climatological periods.

In other words, if your climatological mean data looks like this:
```
          |2010 - 2040 | 2040 - 2070|
------------------------------------
January   | 1          | 3          |
February  | 2          | 4          |
```

The Annual Cycle Graph graphs [1, 2] or [3, 4] depending on which climatology is set on its dropdown, but the Long Term Average Graph graphs [1, 3] or [2, 4] depending on which month is set on its dropdown.

The only "actual time" graph, which is used only for datasets without climatological means, is the TimeSeriesGraph.